### ISSUE TITLE
Build ops for optimizing Value function parameters using fitted Q-learning

### ISSUE THREAD

**Issue by thiagopbueno** [thiagopbueno/model-aware-policy-optimization#9]
Build ops for computing the loss using 1-step bootstrap fitted Q-learning and the update step of gradient descent over the Value function parameters given a TensorFlow optimizer and a batch of experiences.
